dbms/src/Storages/KVStore/Read/LearnerRead.cpp

```diff
@@ -103,17 +105,19 @@ void validateQueryInfo(
             fail_status = status;
             LOG_WARNING(
                 log,
-                "Check after snapshot acquired from storage, region_id={} version={} handle_range={} status={}",
+                "Check after snapshot acquired from storage, region_id={} version={} handle_range={} status={} "
+                "fail_msg={}",
```

Comment:
```suggestion
                "err_msg={}",
```

Reply:
This message is extra, the final error msg is construncted in UnavailableRegions